### DISCUSSION TITLE
Connecting to remote server via Jupyter

### DISCUSSION THREAD

**Discussion by hiraksarkar** [microsoft/vscode-jupyter#16326] [Questions and Answers] [answered]
Hi, 

It seems the documentation at https://code.visualstudio.com/docs/datascience/notebooks-web is out of date as there is no *Jupyter: Specify Jupyter Server for Connections * command any more. Alternatively the kernel picker only works when the Jupyter notebook is opened (which is in a remote cluster in a slurm system). Can't we directly use it just like a browser when the tunnel is established, as we do in browser?

**Comment by DonJayamanne** [comment, marked as answer]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/14476
